tests/secrets.sample.yaml

```diff
@@ -1,10 +1,10 @@
-# Command to login to OpenShift
-oc_login_command: |
-  oc login -u kubeadmin -p CUSTOMIZE_THIS
-
 # Adopted OpenStack admin password. Matching the install_yamls default
 # to reduce developer confusion.
-admin_password: 12345678
+admin_password: 123456789
+
+# Command to login to OpenShift
+oc_login_command: |
+  oc login -u kubeadmin -p {{ admin_password }}
```

Comment:
The admin_password is for OpenStack while this password here is for CRC, i don't think they should be assumed to be the same. In general i didn't think CI would be using this file directly. Rather write a `test/secrets.yaml` file from scratch?